.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Continuous Integration'
+on: [ push, pull_request ]
+
+env:
+  OSSRH_USERNAME: ${{ secrets.SONATYPE_S01_USERNAME }}
+  OSSRH_TOKEN: ${{ secrets.SONATYPE_S01_PASSWORD }}
+  MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_CENTRAL_GPG_PASSWORD }}
+jobs:
+  BuildAndPublish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v3
+        with: # running setup-java again overwrites the settings.xml
```

Comment:
When was it run the first time? I don't get the comment.

Reply:
* this step sets the settings.xml file for maven in GHA